### ISSUE TITLE
WCAG 2.1 AA Research and Documentation

### ISSUE THREAD

**Issue by ArdaYalcindag** [bounswe/bounswe2025group5#330]
### Description
Research should be done about Web Content Accessibility Guidelines (WCAG) 2.1 under WC3 standarts and a documentation should be written about general idea and how the requirements can be changed according to these standarts.

### Type of Update
- [ ] Typo/Grammar Fix
- [ ] Clarification/Explanation Needed
- [x] Add New Information
- [ ] Remove Outdated Information
- [ ] Update Example/Code
- [ ] Other (please specify)

### Acceptance Criteria
- [x] WCAG 2.1 is researched and a brief explanation is written
- [x] A section on how requirements can be updated according to WCAG standarts is written

### Specific Details
- [Wiki page for the documentation](https://github.com/bounswe/bounswe2025group5/wiki/WC3-Standards)
- [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21)

### Additional Notes
@yukseleren should review and close the issue

### Deadline
27.09.2025 - 20.00

**Comment by abdarslan** [comment]
I reviewed your documentation page and found your suggested list of requirements on point. You can make the necessary changes to requirements document and please check if there are any changes to the document to not repeat any enumeration when youre adding these new ones. Good work.
